lib/split/dashboard.rb

```diff
@@ -17,58 +18,78 @@ class Dashboard < Sinatra::Base
     helpers Split::DashboardHelpers
     helpers Split::DashboardPaginationHelpers
 
-    get '/' do
+    get "/" do
       # Display experiments without a winner at the top of the dashboard
       @experiments = Split::ExperimentCatalog.all_active_first
+      @unintialized_experiments = Split.configuration.experiments.keys - @experiments.map(&:name)
 
       @metrics = Split::Metric.all
 
       # Display Rails Environment mode (or Rack version if not using Rails)
-      if Object.const_defined?('Rails')
+      if Object.const_defined?("Rails")
         @current_env = Rails.env.titlecase
       else
         @current_env = "Rack: #{Rack.version}"
       end
       erb :index
     end
 
-    post '/force_alternative' do
+    post "/initialize_experiment" do
+      Split::ExperimentCatalog.find_or_create(params[:experiment]) unless params[:experiment].nil? || params[:experiment].empty?
+      redirect url("/")
+    end
+
+    post "/force_alternative" do
       experiment = Split::ExperimentCatalog.find(params[:experiment])
       alternative = Split::Alternative.new(params[:alternative], experiment.name)
-      alternative.increment_participation
-      Split::User.new(self)[experiment.key] = alternative.name
-      redirect url('/')
+
+      cookies = JSON.parse(request.cookies["split_override"]) rescue {}
+      cookies[experiment.name] = alternative.name
+      response.set_cookie("split_override", { value: cookies.to_json, path: "/" })
+
+      redirect url("/")
     end
 
-    post '/experiment' do
+    post "/experiment" do
       @experiment = Split::ExperimentCatalog.find(params[:experiment])
       @alternative = Split::Alternative.new(params[:alternative], params[:experiment])
       @experiment.winner = @alternative.name
-      redirect url('/')
+      redirect url("/")
     end
 
-    post '/start' do
+    post "/start" do
       @experiment = Split::ExperimentCatalog.find(params[:experiment])
       @experiment.start
-      redirect url('/')
+      redirect url("/")
     end
 
-    post '/reset' do
+    post "/reset" do
       @experiment = Split::ExperimentCatalog.find(params[:experiment])
       @experiment.reset
-      redirect url('/')
+      redirect url("/")
     end
 
-    post '/reopen' do
+    post "/reopen" do
       @experiment = Split::ExperimentCatalog.find(params[:experiment])
       @experiment.reset_winner
-      redirect url('/')
+      redirect url("/")
+    end
+
+    post "/update_cohorting" do
+      @experiment = Split::ExperimentCatalog.find(params[:experiment])
+      case params[:cohorting_action].downcase
+      when "enable"
+        @experiment.enable_cohorting
+      when "disable"
+        @experiment.disable_cohorting
+      end
+      redirect url("/")
```

Comment:
not sure why this is appearing as a diff as this was already added by me in our forked repo ~ maybe since it was added separately to the public repo and being merged back in 🤷‍♂️